src/main.rs

```diff
@@ -333,6 +334,13 @@ struct Barcode<'a>(&'a [u8], &'a [u8]);
 #[allow(clippy::too_many_lines)]
 fn main() -> Result<()> {
     let opts = setup();
+    let progress_logger = ProgLogBuilder::new()
+        .name("fqgrep-progress")
+        .noun("reads")
+        .verb("Searched")
+        .unit(50_000_000)
```

Comment:
Shall we separate with comma's ([there's a crate for that](https://docs.rs/thousands/0.1.4/thousands/trait.Separable.html#method.separate_with_commas))?

Reply:
Sure! I'll put it behind a feature flag I think to keep proglog low dependency by default, but agree tidier numbers would be nice.

Reply:
I am not going to hold this PR up, thank-you @sstadick !